packages/components/src/date-time/timezone.tsx

```diff
@@ -24,11 +25,9 @@ const TimeZone = () => {
 		return null;
 	}
 
-	const offsetSymbol = timezone.offset >= 0 ? '+' : '';
+	const offsetSymbol = Number( timezone.offset ) >= 0 ? '+' : '';
 	const zoneAbbr =
-		'' !== timezone.abbr && isNaN( timezone.abbr )
-			? timezone.abbr
-			: `UTC${ offsetSymbol }${ timezone.offset }`;
+		timezone.abbr || `UTC${ offsetSymbol }${ timezone.offset }`;
```

Comment:
We usually try to introduce the least amount of runtime changes as possible — is there a reason why we're changing this check and not checking for `NaN` anymore?

Reply:
I can't remember exactly what led me here... it was to do with `isNaN` only accepting `number`. 

Looking at it now, I think the better and more minimal way to do this is to use `Number.isNaN` as it accepts any argument type.